src/routes/product-page/product-page.component.jsx

```diff
@@ -0,0 +1,63 @@
+import { gql, useQuery } from "@apollo/client";
+import { useParams } from "react-router-dom";
+import { useState, useEffect } from "react";
+
+import ProductDetail from "../../components/product-detail-container/product-detail-container.component";
+
+import "./product-page.styles.scss";
+
+const GET_PRODUCT_DETAILS = gql`
```

Comment:
Cria uma pasta ou arquivo separado pras queries, deixar elas lá e só importa no arquivo onde for usar